### PR TITLE
Edit Starlark README

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: plugins/processors/starlark/README.md [hunk 1]
```diff
@@ -4,60 +4,69 @@ The `starlark` processor calls a Starlark function for each matched metric,
...
+
+
+#### Example
+
 ```toml
 [[processors.starlark]]
-  ## The Starlark source can be set as a string in this configuration file, or
-  ## by referencing a file containing the script.  Only one source or script
-  ## should be set at once.
-  ##
```
Comment (sjwang90) [inline plugins/processors/starlark/README.md:25]: I would keep the general configuration in there. It's a good reference to have what is the plugin configuration how it is in the `telegraf.conf` as a reference for users to copy & paste from the readme.

File: plugins/processors/starlark/README.md [hunk 2]
```diff
@@ -4,60 +4,69 @@ The `starlark` processor calls a Starlark function for each matched metric,
...
 types and functions are exposed to the script.
 
-**Metric(*name*)**:
+- **Metric(*name*)**:
 Create a new metric with the given measurement name.  The metric will have no
 tags or fields and defaults to the current time.
 
 - **name**:
-The name is a [string][string] containing the metric measurement name.
+The name is a [string] containing the metric measurement name.
```
[the remark this answers is not in the document]
Comment (pierwill) [reply plugins/processors/starlark/README.md:65]: The single `[link]` renders these links correctly [on my branch](https://github.com/pierwill/telegraf/tree/edit-starlark-readme/plugins/processors/starlark).

File: plugins/processors/starlark/README.md (hunk 1 above)
Comment (pierwill) [reply plugins/processors/starlark/README.md:25]: Okay. Will update—

File: plugins/processors/starlark/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielnelson) [reply plugins/processors/starlark/README.md:65]: I've found that some Markdown implementations work without the extra `[]` but technically I believe it is a syntax error.

[the remark this answers is not in the document]
Comment (pierwill) [reply plugins/processors/starlark/README.md:65]: Fair enough! 😇 

https://github.github.com/gfm/#collapsed-reference-link

**Pull request merged**